spec/std/enumerable_spec.cr

```diff
@@ -393,6 +394,11 @@ describe "Enumerable" do
     end
   end
 
+  describe "each_step" do
+    it_iterates "yields the every 2nd element", %w[a c e], %w[a b c d e f].each_step(2)
+    it_iterates "accepts an optional offset parameter", %w[b d f], %w[a b c d e f].each_step(2, offset: 1)
```

Comment:
I'd like to have more specs for some edge cases such as zero values, negative values, offset or step size bigger than the array.